embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/injection/InitModule.kt

```diff
@@ -82,18 +80,15 @@ internal class InitModuleImpl(
         SpansSinkImpl()
     }
 
+    private val openTelemetrySdk: OpenTelemetrySdk by singleton {
+        OpenTelemetrySdk(
+            openTelemetryClock = openTelemetryClock,
+            spanProcessor = EmbraceSpanProcessor(EmbraceSpanExporter(spansSink))
```

Comment:
@nelsitoPuglisi We can configure the SpanProcessor to use a composite `SpanExporter` so we can export to multiple destinations